README.md

```diff
@@ -34,8 +34,17 @@ The current state of the [OCI Artifacts][oci-artifacts] repository:
 
 ## Related Projects Working on Extending OCI Specs
 
-  - [OCI Reference Type Working Group][oci-reftype-wg]
-  - [CNCF ORAS Artifacts Spec][oras-artifacts]
+[OCI Reference Type Working Group][oci-reftype-wg]
+
+Other projects:
+
+- [Docker buildkit][buildkit]
```

Comment:
@sudo-bmitch, can you help us understand the ordering of the displayed list?
If you alpha sort the displayed values, wouldn't it be:

[CNCF distribution](https://github.com/distribution/distribution)
[CNCF ORAS Artifacts Spec](https://github.com/oras-project/artifacts-spec/)
[Docker buildkit](https://github.com/moby/buildkit)
[Google go-containerregistry/crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane)
[OpenSSF sigstore/cosign](https://github.com/sigstore/cosign)
[regclient](https://github.com/regclient/regclient/)
[zot](https://github.com/project-zot/zot)

Reply:
The link isn't what people see.

Reply:
True, but it feels weird to change the order when something gets donated, like Docker Distribution to CNCF Distribution. Do we need to strip off the parent orgs to reduce confusion? I usually just call it Kubernetes and not CNCF Kubernetes.

Reply:
Can you also help understand why this list is added to the readme.md as opposed to [implementations](https://github.com/opencontainers/artifacts/blob/main/implementors.md)

Reply:
I believe that's for implementations of the existing repo, and these were pointers to future development work. I'm sure a bunch of these projects could be listed in both, but this PR is just focused on the projects working on the future development.

Reply:
was why I was asking for a description regarding reason for link here.. a brief comment relating what each project may be doing, normally from the project owners if they would comment and/or approve the mention

Reply:
another .md page would be warranted if it gets tldr

Reply:
i'm fine either way, starting here and updating in a follow up or add just enough explanation here..  to make alpha order obvious I guess just show the string being used..

Reply:
While we're adding things, might as well also add [`ko`](https://github.com/google/ko) since it attaches SBOMs to images by default, using the same mechanism as cosign.

As a contributor to cosign, and maintainer of crane and ko, if we think we need a little text blurb, I can propose:

```
- [cosign][cosign] supports attaching signatures, SBOMs and attestations to images in a manner supported by [many registry implementations](https://github.com/sigstore/cosign#registry-support)
- [crane][crane] supports modifying and moving artifacts between supported registry implementations
- [ko][ko] supports building images for Go applications, including attaching SBOMs using cosign's implementation
```

Reply:
If we don't have a statement from the project owner, should we remove them from the above list? My goal was to be inclusive without triggering a debate over wording of a description that would delay this getting merged.

What specific changes are being requested that would result in the project maintainers approving and merging this?